src/Microsoft.Azure.Relay/HybridConnectionStream.cs

```diff
@@ -15,6 +15,12 @@ namespace Microsoft.Azure.Relay
     /// </summary>
     public abstract class HybridConnectionStream : Stream
     {
+
```

Comment:
If there's a blank line between the '{' and the line with the comment please remove it.  (These cause build errors in our OneBranch system due to StyleCop enabled.)